lib/config/default.js

```diff
@@ -178,5 +178,6 @@ module.exports = {
   //    Generated id:   "31-good-morning-my-friend---do-you-have-5"
   //    2nd appearance: "31-good-morning-my-friend---do-you-have-5-1"
   //    3rd appearance: "31-good-morning-my-friend---do-you-have-5-2"
-  linkifyHeaderStyle: 'keep-case'
+  linkifyHeaderStyle: 'keep-case',
+  autoVersionCheck: true
```

Comment:
add corresponding environment variable

Reply:
This would also need to be in 

https://github.com/hackmdio/codimd/blob/ea52ed86895582139f08562c0dd5bfe2b5cde00a/lib/config/environment.js

But most of all, it must be documented in https://hackmd.io/c/codimd-documentation/%2Fs%2Fcodimd-configuration